sim/core/mana.go

```diff
@@ -33,11 +33,11 @@ type manaBar struct {
 // as well as enable the mana gain action to regenerate mana.
 // It will then enable mana gain metrics for reporting.
 func (character *Character) EnableManaBar() {
-	character.EnableManaBarWithModifier(1.0)
+	character.EnableManaBarWithModifier(1.0, true)
 	character.Unit.SetCurrentPowerBar(ManaBar)
 }
 
-func (character *Character) EnableManaBarWithModifier(modifier float64) {
+func (character *Character) EnableManaBarWithModifier(modifier float64, allowSpellPowerScaling bool) {
```

Comment:
This can be reverted as we discussed in the discord cleaner ways to overwrite the spell power scaling

Reply:
ok